apps/pay/app/lnurlp/[username]/callback/route.ts

```diff
@@ -8,6 +8,8 @@ import { gql } from "@apollo/client"
 
 import Redis from "ioredis"
 
+import originalUrl from "original-url"
```

Comment:
is this required? seems is not maintained (last update 4 years ago)

Reply:
I believe it is necessary in the case that the request is proxied. I will try to bring the logic within the app as I think its pretty simple.

Reply:
> is this required? seems is not maintained (last update 4 years ago)

I think I actually removed this library because of that.

Reply:
It was still present in the repo, but I've now removed it.